presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/CheckSubqueryNodesAreRewritten.java

```diff
@@ -52,7 +53,7 @@ public PlanNode optimize(PlanNode plan, Session session, TypeProvider types, Pla
                     error(lateralJoinNode.getCorrelation(), lateralJoinNode.getOriginSubqueryError());
                 });
 
-        return plan;
+        return new PlanOptimizerOutput(plan, Optional.empty());
```

Comment:
Why this empty?Maybe have a second constructor? Or is it a json issue? Same applies to all the optimizers

Reply:
Oh, currently I am returning empty here, as this PR already touches a lot of files. I will follow up with another PR which adds the logic for each optimizer to populate this information.